app/api/email_utils.py

```diff
@@ -74,3 +74,32 @@ def send_email_verification_message(user_name, email):
     html = render_template('email_confirmation.html', confirm_url=confirm_url, user_name=user_name)
     subject = "Mentorship System - Please confirm your email"
     send_email(email, subject, html)
+
+def send_email_request_notification(reciever_id, sender_id, sender_role, end_date, notes):
+    """Sends a mentorship request message to the specified user.
+    1. send_email() for mentorship_relation/send_request route (mentorship_relation.py file):
+    Finds out sender_id, reciever_id and sender_role. It calls send_email_request_notification() 
+    2. This function retrieves usernames and email addresses of sender and reciever. It 
+    converts timestamp to human readable format and sends email.
+    3. email_confirmation.html provides email template
+
+    Args: reciever_id, sender_id, sender_role, end_date, notes
+    """
+    from app.database.models.user import UserModel
+    from datetime import datetime
+    # retrieve objects based on id
+    reciever = UserModel.find_by_id(reciever_id)
+    sender = UserModel.find_by_id(sender_id)
+    # convert timestamp to datetime
+    dateTime = datetime.fromtimestamp(end_date)
+    html = render_template(
+        'request_notification.html',
+        reciever_name = reciever.username,
+        sender_name = sender.username,
+        sender_email = sender.email,
+        sender_role = sender_role ,
+        end_date = dateTime.strftime("%d %b %Y"), # convert to human readable form "%d %b %Y"
+        notes = notes
+        )
+    subject = "Mentorship System - You have a mentorship request"
```

Comment:
```suggestion
    subject = "Mentorship System - You have a mentorship request!"
``` 
Just a suggestion and not a requirement, it feels exciting having a new request :D 
It's not a mandatory change request, just a suggestion, feel free to not change 😉